src/population.py

```diff
@@ -0,0 +1,58 @@
+import random
+
+class Population():
+    def __init__(self, solutions, fitness):
```

Comment:
`def __init__(self, solutions = None, fitness = None):`

Since when we create an instance of Population class we may not have specific solutions or fitness